eZ/Bundle/EzPublishCoreBundle/Command/NormalizeImagesPathsCommand.php

```diff
@@ -88,19 +89,28 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         $imagesCount = $this->imageGateway->countDistinctImages();
```

Comment:
The block starting here and ending with `$io->progressFinish();` at line 120 could be also encapsulated into a single method for better readability